MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -201,7 +201,7 @@ double GeoMapper::getMeshElevation(
 	    _grid->getNearestPoint(MathLib::Point3d{{{x, y, 0}}});
 	const std::vector<MeshLib::Element*> elements(
 	    _surface_mesh->getNode(pnt->getID())->getElements());
-	GeoLib::Point* intersection(nullptr);
+	std::unique_ptr<GeoLib::Point> intersection;
```

Comment:
Yes. The default ctor constructs a std::unique_ptr that owns nothing. [cppref](http://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr)